homeassistant/components/frontend/__init__.py

```diff
@@ -559,7 +559,7 @@ def websocket_get_themes(hass, connection, msg):
                     "themes": {
                         "safe_mode": {
                             "primary-color": "#db4437",
-                            "accent-color": "#eeee02",
+                            "accent-color": "#e0d202",
```

Comment:
Is this a material design color ?

Reply:
https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=FFEB3B